modules/dev-utils/src/main/java/org/apache/ignite/development/utils/indexreader/IgniteIndexReader.java

```diff
@@ -234,7 +234,9 @@ public IgniteIndexReader(
         outStream = isNull(outputStream) ? System.out : new PrintStream(outputStream);
         outErrStream = outStream;
 
-        idxStore = filePageStoreFactory.createFilePageStoreWithEnsure(INDEX_PARTITION, FLAG_IDX);
+        List<Throwable> idxStoreErrors = new ArrayList<>();
```

Comment:
The index is also a partition, but with a special ID, so you don't need to split the map.
org.apache.ignite.internal.pagemem.PageIdAllocator#INDEX_PARTITION